plrust/src/lib.rs

```diff
@@ -193,6 +193,25 @@ unsafe fn plrust_call_handler(fcinfo: pg_sys::FunctionCallInfo) -> pg_sys::Datum
     }
 }
 
+#[pg_extern]
+fn list_allowed_dependencies<'a>() -> Result<
```

Comment:
I think I'd prefer the public-facing function simply be named `allowed_dependencies`.  It's just a style thing, I suppose, but I prefer function names describe what they return not what they do.